readme.md

```diff
@@ -89,6 +89,7 @@ $ cpy --help
 
   Example
     $ cpy 'src/*.png' dist
+    $ cpy 'src/*.png' '!src/*.ico' dist
```

Comment:
I agree with you. I just copy and paste from upper line. :) I have no idea now. any suggestion?